src/components/App/App.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Clock from '../Clock/Clock';
+
+import './App.scss';
+
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 47,
```

Comment:
```suggestion
    clockName: Math.floor(Math.random() * 100),
```
You can use random name here as well